bank_statement_journal_items/models/bank_statement_items.py

```diff
@@ -0,0 +1,21 @@
+from odoo import models
+
+class BankStatementItems(models.Model):
```

Comment:
```suggestion
class AccountBankStatement(models.Model):
```
Please also rename the file to the model you're modifying (e.g. `account_bank_statement.py`)